cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -4028,6 +4028,48 @@
     </description>
   </property>
 
+  <!-- Retry configuration for spark metrics retrieval -->
+  <property>
+    <name>spark.metrics.strategy.retry.policy.base.delay.ms</name>
+    <value>5000</value>
```

Comment:
5 seconds of initial delay is too long. Should be something like `100`

Reply:
Done